run/idp-sql/Dockerfile

```diff
@@ -23,4 +26,5 @@ COPY . ./
 # webserver, with one worker process and 8 threads.
 # For environments with multiple CPU cores, increase the number of workers
 # to be equal to the cores available.
-CMD exec gunicorn --bind :$PORT --workers 1 --threads 8 main:app
\ No newline at end of file
+# Using ENTRYPOINT to ensure SIGTERM is correctly handled.
+ENTRYPOINT ["gunicorn", "--bind", ":8080", "--workers", "1", "--threads", "8", "main:app"]
```

Comment:
`CMD exec gunicorn --bind :$PORT --workers 1 --threads 8 --timeout 0 main:app` should work here. We prefer using CMD over entrypoint because it's easily over-writable. Adding the `exec` should ensure SIGTERM is handled.